index.html

```diff
@@ -283,50 +283,73 @@
 
 ## People &amp; Data {#people-data}
 
-A <dfn>user</dfn> (also <dfn data-lt="individual">person</dfn> or <dfn>data subject</dfn>) is any natural
-person.
-
-We define <dfn data-lt="data">personal data</dfn> as any information relating to a [=person=] such
-that:
-
-* this [=person=] is identified, directly or indirectly, by reference to an
-  identifier such as a name, email address, an arbitrary identifier or
-  identification number, an online identifier such as an IP address or any
-  identifier attached to a device this [=person=] may be using, phone number,
-  location data, or factors specific to the physical, physiological, genetic,
-  mental, economic, cultural, or social identity of that [=person=], as well as
-  identifiers derived from such data, for instance through hashing; or
-* this [=person=] could reasonably be reidentified from a conjunction of this
-  data with other data; or
-* the data pertains to a group of people such that a [=person=] may find
-  themselves to be the subject of a treatment related to this group, even if the
-  entity carrying out the treatment has no way to identify that [=person=].
+A <dfn>user</dfn> (also <dfn data-lt="individual">person</dfn> or <dfn>data subject</dfn>)
+is any natural person.
+
+We define <dfn data-lt="identifier">identifying data</dfn> as any information relating to a
+[=person=] such that any of the following applies:
+
+1. It references this [=person=] directly, for instance through their name,
+  email address, phone number, or other such traditional denotation.
+* It can identify this [=person=] through an arbitrary string or identification
```

Comment:
Probably number this whole list so that people can easily refer to particular items.
```suggestion
1. It can identify this [=person=] through an arbitrary string or identification
```